homework-classes/Contributor.js

```diff
@@ -14,6 +13,21 @@ class Contributor {
    */
   render(container) {
     // TODO: replace the next line with your code.
-    Util.createAndAppend('pre', container, { text: JSON.stringify(this.contributor, null, 2) });
+    const contributorsDiv = Util.createAndAppend('div', root, { id: 'contributors' });
```

Comment:
Don't think you need this. You can just treat the `container` variable above as the contributorsDiv.

Reply:
ignore